carmin.yaml

```diff
@@ -532,61 +532,8 @@ components:
           type: string
     Pipeline:
       type: object
-      required:
-        - identifier
-        - name
-        - version
-        - properties
-        # - parameters optional in 0.2
-      properties:
-        identifier:
-          type: string
-          format: ascii
-        name:
-          type: string
-        version:
-          type: string
-          format: ascii
-        description:
-          type: string
-        canExecute:
-          type: boolean
-          description: true if the user who requested the pipeline can execute it
-        parameters:
-          type: array
-          items:
-            $ref: '#/components/schemas/PipelineParameter'
-        properties:
-          type: object
-          additionalProperties:
-            type: string
-          description: the properties (as keys) and their values that describe the pipeline. 
-            The properties used must be listed in the "supportedPipelineProperties" of the "getPlatformProperties" method.
-        errorCodesAndMessages:
-          type: array
-          items:
-            $ref: '#/components/schemas/ErrorCodeAndMessage'
-    PipelineParameter:
-      type: object
-      required:
-        - name
-        - type
-        - isOptional
-        - isReturnedValue
-      properties:
-        name:
-          type: string
-          format: ascii
-        type:
-          $ref: '#/components/schemas/ParameterType'
-        isOptional:
-          type: boolean
-        isReturnedValue:
-          type: boolean
-        defaultValue:
-          description: Default value. It must be consistent with the parameter type.
-        description:
-          type: string
+      $ref: 'https://raw.githubusercontent.com/boutiques/boutiques/master/tools/python/boutiques/schema/descriptor.schema.json'
```

Comment:
New schema for Pipelines reduce the expressibility for parameters. There are less parameter types supported (for instance list) and output can only be files (but before the PR it could be any parameter type).

Reply:
Boutiques does support lists. I don't think there are less input types in Boutiques that there are in CARMIN:

| CARMIN types  | Boutiques types    |
-----------------------------------------------------
| Int64                  |   Number - integer |
| Boolean             |   Flag                    |
| String                 | String                     |
| File                    | File                        |
| List                    |  <type> - list          |
| Double              | Number - not integer |
-------------------------------------------------------

 I agree for output types, see suggestion below.

Reply:
Check the size of ints in JSON